.gitignore

```diff
@@ -317,5 +317,5 @@ tpm/evp_tpm
 /ESP32/**/*.sln
 
 # auto-created CMake backups
-**/CMakeLists.txt.old
-
+**/CMakeLists.txt.old
```

Comment:
Did you change anything to this file? If not, please revert the changes.

Reply:
Toppers_app could not be pushed because *app exists.
I changed it to #*app, it went into commit.
I put it back in.
So no change.